web/src/main/java/se/inera/intyg/webcert/web/service/monitoring/MonitoringLogServiceImpl.java

```diff
@@ -426,51 +996,123 @@ public void logSamlStatusForFailedLogin(String issuer, String samlStatus) {
 
     @Override
     public void logTestCertificateErased(String certificateId, String careUnit, String createdUser) {
-        logEvent(MonitoringEvent.TEST_CERTIFICATE_ERASED, certificateId, careUnit, createdUser);
+        try (MdcCloseableMap ignored = MdcCloseableMap.builder()
+            .put(MdcLogConstants.EVENT_ACTION, toEventType(MonitoringEvent.TEST_CERTIFICATE_ERASED))
+            .put(MdcLogConstants.EVENT_TYPE, MdcLogConstants.EVENT_TYPE_DELETION)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_UNIT_ID, careUnit)
+            .put(MdcLogConstants.USER_ID, createdUser)
```

Comment:
Suggest that we remove USER_ID as it is not the user that erases it but instead the user that created the test certificate that is erased. Don't see great need to index this user.

Instead add event.certificate.id

Reply:
done